### ISSUE TITLE
Implement Twitter Bot for Spanish Lessons

### ISSUE THREAD

**Issue by walshbr** [programminghistorian/jekyll#471]
Right now, the Twitter bot only sends out English tweets twice a week. There are a few steps required to get it working for Spanish:

- [x] Update technical side of the bot to accommodate a second tab from the [Google spreadsheet](https://docs.google.com/spreadsheets/d/1o-C-3WwfcEYWipIFb112tkuM-XOI8pVVpA9_sag9Ph8/edit#gid=1625380994) storing the tweets.
- [ ] Populate that second tab of the spreadsheet with Spanish tweets.
- [x] Translate the [addition to the editorial workflow](https://github.com/programminghistorian/jekyll/pull/464) that informs new authors about how to make additions to the tweet queue.

I held out on accommodating this feature at the time because the Spanish team felt there weren't enough Spanish translations to warrant regular tweeting in this way. So just let me know when you're ready to revisit this. Then I'll finish out step one so that you can proceed with steps two and three.

**Comment by arojascastro** [comment]
Thanks @walshbr. Hope to put some effort on this as soon as possible. I understand we just need to replicate that tab and populate with messages in Spanish that take care of our translations, right? I think we may need extra help for this. Do you want to collaborate? @vgayolrs @mariajoafana

**Comment by walshbr** [comment]
Yep! That's correct. I just made the extra tab for you with the appropriate column headings. You can feel free to just translate the tweets that I have listed for English, if that is easier. I'm perfectly aware that the content there could be improved, and I'd welcome you doing it if you want to generate new, Spanish-specific tweets. But it's a lot of content to generate. So I'd do whatever is easiest for you.

I just tested, and the presence of that second tab and stuff in it won't affect the regular English tweets that I've set up. So you should be fine to just start populating the Spanish tab without fear and without waiting on me to implement the technical side of the process. Your work won't go public until we're ready. Just leave column E blank, as that's the one used by the bot to track its progress through the lessons. Let me know if you have other questions or if I can do anything else!

**Comment by arojascastro** [comment]
I have populated the tab with messages for 7 lessons. Do you want to try @walshbr? Hope @mariajoafana and @vgayolrs can help me to translate the messages on Python series in the following days.

Can we add messages asking for contributions and collaborators? The link should be the home page, or the collaborate page.

**Comment by walshbr** [comment]
Cool! I can use what you've got here to make sure the Spanish bot functions. 

And that's a great idea about soliciting contributions and collaborators. I'll make a separate ticket for it. I'll need to modify the functionality slightly so that it tweets those links from a different queue. Were we to just drop those tweets in the present queues, they would be cycled through with everything else and only come up once a year or so.

**Comment by walshbr** [comment]
@arojascastro - got the bot working now so that it is all set up to start tweeting Spanish messages whenever you're ready. A couple questions:

1. Did you want to add it to the queue now or wait? We could just start it now if you would like - we don't need to wait until the spreadsheet is completed to do so. After all, the bot sits on a single row for a week. So you've already got enough content there to fill out the next couple months.

2. What would be good times/days of the week for the Spanish readership? Right now, the English tweets are going out at:

Monday - 9AM PST, 12PM EST, 4PM London time
Thursday 6:30 AM PST, 9:30 AM EST, 1:30PM London

Since the Spanish translations are in a different spreadsheet, they're going to be running on their own randomized selector. And they're not fully populated yet anyway, so not sure it makes a lot to directly translate the English tweets right away. I guess I'm inclined to consider the Spanish messages as their own recurring sequence - Tuesday and Wednesday or Tuesday and Friday, at times that make sense to you. 

Let me know what you think, and I can either schedule those recurring tweets right away or make a note to do so whenever you all are ready.

**Comment by arojascastro** [comment]
Hi @walshbr, thanks a lot for your work. I am inclined to start using the bot if @mariajoafana and @vgayolrs agree. I would go with those same times so they are published during the day in Europe (let's say  Tuesday) and Latin America (Wednesday). What do you think @mariajoafana and @vgayolrs ?

**Comment by mariajoafana** [comment]
Hi! I just see this. I think that the times are fine and I agree on starting to use the bot right away. I'll also help populate the spreadsheet with new tweets.

**Comment by walshbr** [comment]
Sounds great! I just flipped the switch, so the bot should be sending the Spanish tweets out at those times Tuesday and Wednesday. I've been keeping an eye on the twitter stream to make sure the bot is functioning as expected, but let me know if anything seems weird or off.

One thing to note when putting together this spreadsheet is that @acrymble felt like it was a bit much for the bot to tweet out each individual lesson from that last sequence of Python tutorials at the bottom of the lessons page. So I've lumped those all in under one umbrella tweet at ID number 45.
